Source/CarthageKit/GitHub.swift

```diff
@@ -249,12 +322,12 @@ private func loadCredentialsFromGit() -> SignalProducer<BasicGitHubCredentials?,
 }
 
 
-internal func loadGitHubAuthorization() -> SignalProducer<String?, CarthageError> {
+internal func loadGitHubAuthorization(forURL URL: GitHubRepository.URL) -> SignalProducer<String?, CarthageError> {
 	let environment = NSProcessInfo.processInfo().environment
 	if let accessToken = environment["GITHUB_ACCESS_TOKEN"] as? String {
```

Comment:
It currently does not support the GitHub access token for GitHub Enterprise repositories (originally added in #605). I have no idea to make it successful now.